olGO/Extensions/JSONDecoder+olGO.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  JSONDecoder+olGO.swift
+//  olGO
+//
+//  Created by Zach Eriksen on 4/11/20.
+//  Copyright © 2020 oneleif. All rights reserved.
+//
+
+import Foundation
+
+extension JSONDecoder {
+    static var dateDecoder: JSONDecoder = {
+        let formatter = DateFormatter()
```

Comment:
Can't we just use `ISO8601DateFormatter` here?

```swift
let formatter = ISO8601DateFormatter()
formatter.formatOptions =  [.withInternetDateTime, .withFractionalSeconds]
```

Reply:
Getting this error:
```
Cannot convert value of type 'ISO8601DateFormatter' to expected argument type 'DateFormatter'
```